### PR TITLE
Added email and Slack notifications for MTAlignment and MTCom.

### PR DESCRIPTION
**Pull request by rbovill**

### CODE REVIEW

**Review by dmills-lsst — APPROVED** [review]
ok

File: src/org/lsst/ts/jenkins/components/Csc.groovy [hunk 1]
```diff
@@ -36,6 +36,8 @@ def email() {
                 "ts-integrationtests": "rbovill@lsst.org",
                 "ts-linearstage": "ecoughlin@lsst.org",
                 "ts-m2": "tribeiro@lsst.org",
+                "ts-mtalignment": "dmills@lsst.org, pkubanek@lsst.org, rowen@uw.edu",
+                "ts-mtcom": "ttsai@lsst.org",
```
Comment (teweitsai) [inline src/org/lsst/ts/jenkins/components/Csc.groovy]: `ts-m2com` instead of `ts-mtcom`

File: src/org/lsst/ts/jenkins/components/Csc.groovy [hunk 2]
```diff
@@ -103,6 +106,8 @@ def slack_id() {
                 "ts-integrationtests": rob,
                 "ts-linearstage": eric,
                 "ts-m2": tiago,
+                "ts-mtalignment": petr,
+                "ts-mtcom": tewei,
```
Comment (teweitsai) [inline src/org/lsst/ts/jenkins/components/Csc.groovy]: `ts-m2com` instead of `ts-mtcom`

File: src/org/lsst/ts/jenkins/components/Csc.groovy [hunk 3]
```diff
@@ -67,6 +69,7 @@ def slack_id() {
     String rob = "U2NF4GWV8"
     String russell = "U2JPAP0F6"
     String tiago = "U72CH91L2"
+    String tewei = "U3F32NNJ2"
```
Comment (teweitsai) [inline src/org/lsst/ts/jenkins/components/Csc.groovy:72]: How is the "U3F32NNJ2" decided?

**Review by teweitsai — APPROVED** [review]
Looks good. It is `ts-m2com` instead of `ts-mtcom`. Thanks!

File: src/org/lsst/ts/jenkins/components/Csc.groovy (hunk 1 above)
Comment (rbovill) [reply src/org/lsst/ts/jenkins/components/Csc.groovy]: fixed

File: src/org/lsst/ts/jenkins/components/Csc.groovy (hunk 2 above)
Comment (rbovill) [reply src/org/lsst/ts/jenkins/components/Csc.groovy]: fixed

File: src/org/lsst/ts/jenkins/components/Csc.groovy (hunk 3 above)
Comment (rbovill) [reply src/org/lsst/ts/jenkins/components/Csc.groovy:72]: It comes from Slack itself.

**Pull request merged**